crmsh/xmlutil.py

```diff
@@ -1450,14 +1450,12 @@ def nvpair_id(nvpairid, name, value):
     return new("nvpair", id=nvpairid, name=name, value=value)
 
 
-def nvpair_ref(idref, name=None):
+def nvpair_ref(idref):
     """
-    <nvpair id-ref=<idref> [name=<name>]/>
+    <nvpair id-ref=<idref>/>
```

Comment:
Indeed pacemaker-3.0 dropped the support for nvpair name with id-ref from the XML schema "pacemaker-4.0". But in case it's existing when an old version syntax of CIB is used, pacemaker interprets it internally.

It makes sense to drop the usage from the manpage/helptext. But crmsh should better still be able to show it in case it is existing.

Reply:
Then I keep that code unchanged, otherwise it wouldn't parse the old syntax
But I drop the syntax mention of '@\<id\>:name' in crm.8.adoc, in `help AttributeReferences`